test/index.js

```diff
@@ -117,6 +123,9 @@ describe("JavaScript components", function() {
     it("should load ZenPlayer from everything.js", function() {
         assert.ok(browser.evaluate("ZenPlayer"));
     });
+    it("should make all requests over https, not http", function() {
+        assert.strictEqual(-1, _js.indexOf("http://", "Please use HTTPS for all scripts"));
```

Comment:
There is a `)` out of place here. There 3 arguments for `assert.strictEqual()` should be:

1. `-1`
2. `_js.indexOf("http://")`
3. `"Please use HTTPS for all scripts"`